test/libsolidity/semanticTests/externalSource/source_name_starting_with_dots.sol

```diff
@@ -0,0 +1,22 @@
+==== ExternalSource: /Users/alex/git/solidity/cmake-build-debug/test/tools/a.sol=_source_name_starting_with_dots/dot_a.sol ====
+==== ExternalSource: /Users/alex/git/solidity/cmake-build-debug/test/b.sol=_source_name_starting_with_dots/dot_dot_b.sol ====
```

Comment:
Why `/Users/alex/git/solidity/cmake-build-debug/`? I don't see any test importing a path like that.

Reply:
oh wow! I think that was introduced by an expectation update. Need to check why that happened.

Reply:
Ah, so that's how it got here :)
I suspect that it's because of `weakly_canonical()`. It converts the path to an absolute one if it can.